### ISSUE TITLE
Too many positinal args reported on `hashlib.shake_128().hexdigest(n)`

### ISSUE THREAD

**Issue by tushar-deepsource** [PyCQA/pylint#5565]
### Bug description

```python
'''test'''
import hashlib

hashlib.shake_128(b'abcd').hexdigest(4)
```

### Command used

```shell
pylint mytest.py
```

### Pylint output

```shell
************* Module mytest
mytest.py:4:0: E1121: Too many positional arguments for method call (too-many-function-args)
```

### Expected behavior

No issues.

https://docs.python.org/3/library/hashlib.html#shake-variable-length-digests States that `shake_128` and `shake_256` return different objects, whose `hexdigest()` function takes in a length parameter.

I did some digging and found that all other objects return a `_hashlib.HASH` object while `shake_128` and `shake_256` return a `_hashlib.HASHXOF` object, both of which are defined in C.

### Pylint version

```shell
pylint 2.11.1
astroid 2.8.6-dev0
Python 3.10.0 (default, Oct 13 2021, 06:45:00) [Clang 13.0.0 (clang-1300.0.29.3)]
```

### OS / Environment

MacOS monterey

**Comment by jacobtylerwalls** [comment]
Dupe of #4039
